rfcs/20221124-monorepo.md

```diff
@@ -0,0 +1,179 @@
+↖️ Table of Contents
+
+# RFC: Monorepo
+
+**Status:** 🚧 WIP, comments are welcome nonetheless
```

Comment:
Is this still accurate? Or can we consider this complete? I would suggest if it's ready we add a completion date that review should be done by here and in the PR description. Moving forward we'll use a two-week deadline. This has already been open for 12-ish days, as I recall, but some folks are starting to share concerns, so I would suggest December 14th as a due date allowing for sufficient time for additional feedback.